AdyenCard/Form/FormCardNumberItem.swift

```diff
@@ -165,10 +211,44 @@ internal final class FormCardNumberItem: FormTextItem, AdyenObserver {
         )
     }
     
+    /// Calculates the length of the string being replaced
+    ///
+    /// e.g. if the range is `2` characters long and the replacementString is `1` character the replacementStringLength would be `-1`
+    private func replacementStringLength(
+        range: NSRange,
+        replacementString: String,
+        in text: String,
+        isDeletingSingleCharacter: Bool
+    ) -> Int {
+        // Special case to allow "deleting" a space
+        // (can only be triggered when the user manually moves the cursor)
+        //
+        // 1234 5678 |310 // Deleting a character
+        if range.length == 1, replacementString.isEmpty, isDeletingSingleCharacter {
```

Comment:
Hmm nice! What about deleting multiple chars at once by dragging/selecting?

Reply:
That already works - this was just a fix for a very edge-case where the user moves the cursor in front of a comma.